src/layouts/NavbarWrapper.tsx

```diff
@@ -1,11 +1,27 @@
-import React from 'react'
+import React, { useState } from 'react'
 import { Navbar } from '@/utils/navbar'
 
-export default function NavbarWrapper( { children } ) {
+
+export default function NavbarWrapper({ children }) {
+
+  const [isEaster, toggleEaster] = useState(false);
+
+  const changeTheme = () => {
+    toggleEaster(isEaster => !isEaster)
+  }
```

Comment:
One classic coding/designing thing is "how do we make this more generic?" in this case, can we make it easy to add more themes without needing a bunch of new methods, props, if statements, etc

Reply:
Yeah, I agree. I've been making things way too generic in my other projects, and one of the suggestions from the ruby object-oriented book was basically "solve the problem at hand" and then worry about refactoring when it comes up. But the other half of that is always having one eye on ease of extensibility later.... you mean adding a new method for every holiday isn't a good or realistic coding practice?

Reply:
lol okay YES actually that is a great point. happens so much that people try to make things "future-proof" without any reason to suspect they will change. in this case i guess i can so easily see us adding another theme? plus i think writing the code in a way which uses a theme ID or something probably? wouldn't end up being much more code/complexity, so I guess that's why in this specific instance it screamed "abstract it!!" to me. but again, your point is solid, and also I would be fine to merge as-is and we can use it as a test to see how long it takes for us to want to refactor lol